server/services/inductionService.ts

```diff
@@ -19,18 +21,28 @@ export default class InductionService {
 
   async getEducationAndTraining(prisonNumber: string, token: string): Promise<EducationAndTraining> {
     try {
-      const induction = await this.getInduction(prisonNumber, token)
+      const induction = await this.retrieveInduction(prisonNumber, token)
       return toEducationAndTraining(induction)
     } catch (error) {
       return { problemRetrievingData: true } as EducationAndTraining
     }
   }
 
+  async getInduction(prisonNumber: string, token: string): Promise<InductionDto> {
```

Comment:
This is about the only part that has not been copied from CIAG UI (though it is based on their version)
We will need a service method to retrieve an Induction and map it into an `InductionDto`, and we will need it for pretty much every story that we are about to get into re: Updating the Induction; so it made sense to add it now.

We previously has a private method called `getInduction`, so I renamed that one to `retrieveInduction`, and then wrote this version of `getInduction`

Reply:
I found this a bit confusing initially. How about putting the error handling in the client and then I think you can get rid of `retrieveInduction()`?